security/pkg/nodeagent/sds/monitoring.go

```diff
@@ -15,150 +15,43 @@
 package sds
 
 import (
-	"github.com/prometheus/client_golang/prometheus"
+	"istio.io/pkg/monitoring"
 )
 
+// TODO: Create an endpoint in Citadel agent to expose metrics.
+// Previously exposed metrics: pending_push_per_connection, stale_conn_count_per_connection,
+// pushes_per_connection, push_errors_per_connection, pushed_root_cert_expiry_timestamp, pushed_server_cert_expiry_timestamp
 var (
 	// totalPushCounts records total number of SDS pushes since server starts serving.
-	totalPushCounts = prometheus.NewCounter(prometheus.CounterOpts{
-		Namespace: "citadel_agent",
-		Subsystem: "sds_service",
-		Name:      "total_pushes",
-		Help:      "The total number of SDS pushes.",
-	})
+	totalPushCounts = monitoring.NewSum(
+		"total_pushes",
```

Comment:
Wait - total_pushes without any prefix ? What about XDS pushes ? The original metric had sds_service as subsystem at least.